tests/failpoints/mod.rs

```diff
@@ -10,9 +10,13 @@ extern crate slog_global;
 mod cases;
 
 fn setup<'a>() -> fail::FailScenario<'a> {
-    let guard = fail::FailScenario::setup();
+    fail::FailScenario::setup()
+}
+
+// The prefix "_" here is to guarantee running this case first.
+#[test]
+fn _ci_setup() {
     test_util::setup_for_ci();
```

Comment:
How about `Once.call_once(test_util::setup_for_ci)`?

Also, could you call `setup` twice in `test_setup`?

Reply:
Did you mean to put `Once.call_once(test_util::setup_for_ci)` to `setup()` and no need to have `_ci_setup()`?